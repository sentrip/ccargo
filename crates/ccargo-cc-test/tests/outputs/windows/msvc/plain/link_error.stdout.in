warn.c
main.c
Generating Code...
main.obj : error LNK2005: func already defined in warn.obj
build\main.exe : fatal error LNK1169: one or more multiply defined symbols found
