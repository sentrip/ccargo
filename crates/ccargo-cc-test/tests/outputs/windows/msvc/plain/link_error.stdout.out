error LNK2005: func already defined in warn.obj
   --> main.obj 
error LNK1169: one or more multiply defined symbols found
   --> build\main.exe 
